docs/_docs/clustering/network-configuration.adoc

```diff
@@ -49,10 +49,18 @@ tab:C++[unsupported]
 The following table describes some most important properties of `TcpDiscoverySpi`.
 You can find the complete list of properties in the javadoc:org.apache.ignite.spi.discovery.tcp.TcpDiscoverySpi[] javadoc.
 
+[CAUTION]
+====
+You should assign `IgniteConfiguration.localHost` or `TcpDiscoverySpi.localAddress`.
```

Comment:
Vlad,

Good explanation, crystal clear and straight to the point. I made some tweaks. Please use this final version below and see if we need to mention `TcpCommunicationSpi.localAddress` along with `TcpDiscoverySpi.localAddress`

_You should initialize the `IgniteConfiguration.localHost` or `TcpDiscoverySpi.localAddress` parameter only with those network interfaces that will be used for inter-node communication. By default, a node binds to and listens on all available IP addresses of an environment it's running on. It can prolong node failures detection if some of the node's addresses are not reachable from other cluster nodes._

Reply:
Fixed with the exception we can bind only to one net. interface currently. Or automatically to all.